redi-connect-front/src/components/molecules/Editable.tsx

```diff
@@ -1,43 +1,49 @@
 import React, { useState } from 'react'
-import { Element } from 'react-bulma-components'
+import { Heading } from 'react-bulma-components'
+import classnames from 'classnames'
 import './Editable.scss'
 
 interface Props {
   title: string
   onSave: Function
   read: React.ReactNode
   children: React.ReactNode
+  className?: string
+  savePossible?: boolean
 }
 
 function Editable (props: Props) {
   const {
     title,
     children,
     read,
-    onSave
+    onSave,
+    savePossible,
+    className
   } = props
 
   const [editable, setEditable] = useState(false)
```

Comment:
Let's pick a better name here. The form values are "always" editable, it's rather a question of whether they are being edited or not.

`isEditing`, `setIsEditing` should do the trick

Reply:
How was it? Naming is a b..... 
Sure, good idea

Reply:
The hardest problem in computer science ;)